pkg/daemon/ceph/client/command.go

```diff
@@ -68,7 +68,8 @@ func FinalizeCephCommandArgs(command string, clusterInfo *ClusterInfo, args []st
 
 	// If the command should be run inside the toolbox pod, include the kubectl args to call the toolbox
 	if RunAllCephCommandsInToolboxPod != "" {
-		toolArgs := []string{"exec", "-i", RunAllCephCommandsInToolboxPod, "-n", clusterInfo.Namespace, "--", "timeout", "15", command}
+		toolArgs := []string{"exec", "-i", RunAllCephCommandsInToolboxPod, "-n", clusterInfo.Namespace, "--",
```

Comment:
This timeout only applies to the ceph commands that are running in the integration tests. This will not apply to the ceph commands running in the operator. So if the intention is for all ceph commands in the operator to timeout after 15s, we likely need another change, otherwise no timeout is added for ceph commands running in the operator.

Reply:
If the intention is just to consistently use 15s instead of a minute timeout in other places, then let's just change the constants and no need to change other code.

Reply:
> This timeout only applies to the ceph commands that are running in the integration tests.

Yes, I understand. My intention in this diff is to clarify the timeout is same as [ceph command](https://github.com/rook/rook/blob/f6bd25ae01c2f295d602fc16c9d1eba11b9edfc0/pkg/daemon/ceph/client/command.go#L65-L67).

> If the intention is just to consistently use 15s instead of a minute timeout in other places, then let's just change the constants and no need to change other code.

OK, I'll change "cephCmd.RunWithTimeout(applyDashboardKeyTimeout)" to `cephCmd.RunWithTimeout(CephConnectionTimeout)` and remove `applyDashboardKeyTimeout`.

I was going to work on unifying two timeout values (15s and 1m) after this PR. Is there any strong reason why there are two timeout values? If not, I'd also like to fix it in this PR. In other words, I'll replace `CmdExecuteTimeout` with `CephConnectionTimeout`.

Reply:
A consistent timeout of 15s sounds good to me, I don't recall a good reason for the separate 1m timeout